src/OpenTelemetry/Trace/Export/SimpleSpanProcessor.cs

```diff
@@ -27,6 +27,7 @@ public class SimpleSpanProcessor : SpanProcessor
     {
         private readonly SpanExporter exporter;
         private bool disposed = false;
+        private int spanCount = 0;
```

Comment:
It's probably unlikely, but concurrent calls could cause increments to be lost and if we ever reached int32.max it would overflow (2147483647).

Instead we could use a ulong wrapped with Interlocked.Exchange that resets to 0 if value == ulong.max.

I guess it depends on how careful / guarded we need to be with SimpleSpanProcessor.